testing_and_setup/compass/ocean/drying_slope/marsh_flooding/100m/comparison.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env python
```

Comment:
I believe you can do this as a sym-link unless you've somehow modified this script from its original values.  If so, probably should generalize the other script into more modular code.